internal/cmd/skupper/system/nonkube/system_delete.go

```diff
@@ -123,8 +124,10 @@ func (cmd *CmdSystemDelete) Run() error {
 			err := cmd.siteHandler.Delete(site.Name)
 			if err != nil {
 				slog.Error("Error while deleting site %q: %s", site.Name, err)
+			} else {
+				crDeleted = true
```

Comment:
It is not yet producing the expected result, as the Delete method is not returning
an error when the provided file does not exist.
Maybe, if not too much, a generic validation (os.Stat for example) could be included
as part of `BaseCustomResourceHandler.DeleteFile` ?